Source/WebKit/Shared/WebCoreArgumentCoders.serialization.in

```diff
@@ -8236,3 +8236,11 @@ enum class WebCore::CrossOriginMode : bool
 [CreateUsing=fromRawString] class WebCore::PublicSuffix {
     String string();
 };
+
+header: <WebCore/CryptoKey.h>
+[CustomHeader] struct WebCore::WrappedCryptoKey {
+    std::array<uint8_t, 24> wrappedKEK;
+    Vector<uint8_t> encryptedKey;
+    std::array<uint8_t, 16> tag;
+};
+
```

Comment:
Super nit:  Don't need blank line at the end of the file.  (Not required to land PR.)

Super nit # 2:  I thought this made sense to put closer to where it landed on the branch rather than the very end.  (Again, not needed to land PR.)